### ISSUE TITLE
(502) Bad Gateway on Download-AppxFromStore.ps1

### ISSUE THREAD

**Issue by muffpistol** [MattiasC85/Scripts#2]
I am receiving a "(502) Bad Gateway" error only when the script tries to download the .blockmap file of a free store app (in this case, Microsoft Camera).  It is able to download the other .appx and .appxbundle files without issue.  
<img width="621" alt="Download-AppxFromStoreError" src="https://user-images.githubusercontent.com/49566160/96006624-af18cf80-0e0b-11eb-9a36-fe3d9072920e.PNG">

**Comment by MattiasC85** [comment]
First and foremost, sorry for the late answer.
After 28 months of saying "I will get that haircut this week" that is now finally done and I'll try to catch up on these other things as well :)

That beeing said, I can't reproduce the issue at hand. :(
What's the storeurl of that you're trying to download ? 
![image](https://user-images.githubusercontent.com/44426149/98749461-0171ef80-23bc-11eb-86b2-d7eb11d5016b.png)

**Comment by muffpistol** [comment]
The store URL is as seen in the screenshot below.  Unfortunately I cannot even get the file list for this particular app (Windows Camera) anymore so I cant confirm your results, though I was able to at least get the file list back when I posted this as an issue.  Are you able to pull the file list again like you did in your reply?  I tried a few other apps and some work while others do not.  I can only assume this is an issue with store.rg-adguard.net and what they are able to access.  For now, I have Global Admin access in Azure so I am able to access the store and download the files that I need online.  I was hoping to use your script as it provides a quick and easy method but for now I guess I will have to use the online method.  Thanks for getting back to me.   

<img width="646" alt="WindowsCameraError" src="https://user-images.githubusercontent.com/49566160/99312598-479bd880-282c-11eb-949d-7d3ddbc7c79a.PNG">
